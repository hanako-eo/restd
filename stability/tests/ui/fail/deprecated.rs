#![feature(staged_api)]
#![stable(feature = "root", since = "1.0.0")]
#![allow(internal_features)]

#[stability::deprecated(feature = "test")]
fn test() {}

fn main() {}
