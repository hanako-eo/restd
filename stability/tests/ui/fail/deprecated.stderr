error: since, note and suggestion need to be set
 --> tests/ui/fail/deprecated.rs:5:1
  |
5 | #[stability::deprecated(feature = "test")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `stability::deprecated` (in Nightly builds, run with -Z macro-backtrace for more info)

error: unrecognized argument
 --> tests/ui/fail/deprecated.rs:5:25
  |
5 | #[stability::deprecated(feature = "test")]
  |                         ^^^^^^^^^^^^^^^^
